Practice/index.html

```diff
@@ -30,42 +23,44 @@
 
  -->
-    <!-- 
-   Link to custom stylesheet
-  -->
-    <link rel="stylesheet" href="./assets/css/style.css" />
-  </head>
+  <link rel="stylesheet" href="./assets/css/style.css" />
+</head>
 
-  <body>
-    <!-- 
-      The links / CDNS to allow the usage of Bootstrap components with interactions (like modal/ navigation bar)
+<body>
+  <!-- add a nav bar -->
+  <!--The links / CDNS to allow the usage of Bootstrap components with interactions (like modal/ navigation bar)
      -->
-    <script
-      src="https://code.jquery.com/jquery-3.5.1.slim.min.js"
-      integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj"
-      crossorigin="anonymous"
-    ></script>
+  <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js"
+    integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj"
+    crossorigin="anonymous"></script>
 
-    <script
-      src="https://cdn.jsdelivr.net/npm/popper.js@1.16.1/dist/umd/popper.min.js"
-      integrity="sha384-9/reFTGAW83EW2RDu2S0VKaIzap3H66lZH81PoYlFhbGU+6BZp6G7niu735Sk7lN"
-      crossorigin="anonymous"
-    ></script>
+  <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.1/dist/umd/popper.min.js"
+    integrity="sha384-9/reFTGAW83EW2RDu2S0VKaIzap3H66lZH81PoYlFhbGU+6BZp6G7niu735Sk7lN"
+    crossorigin="anonymous"></script>
 
-    <script
-      src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"
-      integrity="sha384-B4gt1jrGC7Jh4AgTPSdUtOBvfO8shuf57BaghqFfPlYxofvL8/KUEfYiJOMMV+rV"
-      crossorigin="anonymous"
-    ></script>
+  <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"
+    integrity="sha384-B4gt1jrGC7Jh4AgTPSdUtOBvfO8shuf57BaghqFfPlYxofvL8/KUEfYiJOMMV+rV"
+    crossorigin="anonymous"></script>
+
+
+  <h1 class="animate__animated animate__bounce"> Hello!!!!!!!!</h1>
+  <h2 class="animate__animated animate__zoomInLeft"> Every day, we use zoom! </h2>
+  <h3 class="animate__animated animate__zoomOut"> Every day, we use zoom! </h3>
+  <h4 class="animate__animated animate__zoomInDown"> Every day, we use zoom! </h4>
+  <h5 class="animate__animated animate__zoomInUp"> Every day, we use zoom! </h5>
+  <h6 class="animate__animated animate__zoomInDown"> So Every day, we are consumed! </h6>
 
-    <!-- 
+
+
+  <!-- 
        use CSS to achieve smooth scroll  OR  CDN to smooth scroll, allows you to activate smooth scroll within the application, from this github repo
       https://github.com/cferdinandi/smooth-scroll 
      -->
-    <script src="https://cdn.jsdelivr.net/gh/cferdinandi/smooth-scroll@15/dist/smooth-scroll.polyfills.min.js"></script>
-    <!-- 
+  <script src="https://cdn.jsdelivr.net/gh/cferdinandi/smooth-scroll@15/dist/smooth-scroll.polyfills.min.js"></script>
+  <!-- 
       Script to custom JavaScript
      -->
-    <script src="./assets/javascript/script.js"></script>
-  </body>
-</html>
+  <script src="./assets/javascript/script.js"></script>
+</body>
+
+</html>
```

Comment:
You made good use of animate CSS. Well done. Make sure to place the script tag at the end of your body tag, just before the closing body tag